src-docs/openstack_manager.md

```diff
@@ -167,3 +166,32 @@ __init__(
 
 
 
+---
+
+<a href="../src/openstack_cloud/openstack_manager.py#L242"><img align="right" style="float:right;" src="https://img.shields.io/badge/-source-cccccc?style=flat-square"></a>
+
+## <kbd>class</kbd> `BuildImageConfig`
+The configuration values for building openstack image. 
+
+Attrs:  arch: The image architecture to build for.  base_image: The ubuntu image to use as image build base.  proxies: HTTP proxy settings. 
```

Comment:
this should be `Attributes` so that it renders correctly 🙂

Reply:
Updated, thank you!